lib/screens/preferences/wallet_config.dart

```diff
@@ -97,34 +97,49 @@ class WalletConfigState extends State<WalletConfig> {
     }
   }
 
+  void _clearGeneratedXprv() {
+    setState(() {
+      newXprv = null;
```

Comment:
we need to clear the `xprv` and reset `showXprv` to false. 
```suggestion
      xprv = null;
      newXprv = null;
      showXprv = false;
```